odoo/addons/base/views/res_users_views.xml

```diff
@@ -527,5 +533,147 @@
+                    <header>
+                        <button type="object" class="btn btn-primary" name="revoke" string="Revoke session"/>
+                    </header>
+                    <field name="user_id"/>
+                    <field name="name" class="fw-bold" decoration-success="is_current"/>
+                    <field name="ip_address"/>
+                    <field name="first_activity"/>
+                    <field name="last_activity"/>
+                    <field name="is_current" optional="hidden"/>
+                    <field name="on_disk" optional="hidden"/>
+                    <button type="object" name="revoke" string="Revoke" class="btn btn-secondary" invisible="not on_disk"/>
+                </tree>
+            </field>
+        </record>
+        <record model="ir.ui.view" id="res_users_device_view_kanban">
+            <field name="name">res.users.device.kanban</field>
+            <field name="model">res.users.device</field>
+            <field name="arch" type="xml">
+                <kanban default_order="last_activity desc">
+                    <field name="id"/>
+                    <field name="name"/>
+                    <field name="device_type"/>
+                    <field name="address"/>
+                    <field name="last_activity_display"/>
+                    <field name="is_current"/>
+                    <templates>
+                        <t t-name="kanban-box">
+                            <div class="oe_kanban_card oe_kanban_global_click">
+                                <div class="o_kanban_card_content d-flex">
+                                    <div>
+                                        <t t-if="record.device_type.raw_value === 'laptop'">
+                                            <span class="fa fa-laptop fs-1" title="Laptop" role="img" aria-label="Laptop"/>
+                                        </t>
+                                        <t t-else="">
+                                            <span class="fa fa-mobile fs-1" title="Mobile" role="img" aria-label="Mobile"/>
+                                        </t>
+                                    </div>
+                                    <div class="oe_kanban_details d-flex flex-column ms-3">
+                                        <div class="d-flex align-items-center">
+                                            <strong class="o_kanban_title">
+                                                <field name="name"/>
+                                            </strong>
+                                            <t t-if="record.is_current.raw_value">
+                                                <span class="ms-2 text-success o_status o_status_green"></span>
+                                            </t>
+                                        </div>
+                                        <field name="address"/>
+                                        <field name="last_activity_display"/>
```

Comment:
```suggestion
                                        <t t-out="luxon.DateTime.fromISO(record.last_activity.raw_value).toRelative()"/>
```
You might need to ask a JS framework expert if this is allowed, because I did not see a single occurence in the code to this one exactly, but that works and it avoids the compute field `last_activity_display`